telegram-bot-core/src/main/kotlin/io/github/dehuckakpyt/telegrambot/handling/BotHandling.kt

```diff
@@ -149,5 +181,6 @@ class BotHandling internal constructor(
      *
      * @return if exists then object of selected class else null
      */
-    fun <T : Any> GeneralContainer.transferredOrNull(clazz: KClass<T>): T? = contentConverter.fromContentOrNull(content, clazz)
+    @Suppress("UNCHECKED_CAST")
+    fun <T : Any> GeneralContainer.transferredOrNull(clazz: KClass<T>): T? = contentConverter.fromContentOrNull(content, clazz) ?: nextStepInstance as? T
```

Comment:
I have a feeling something's not right here. I need to think about it

For example, it will not work correctly here if we do 

```kotlin
  command(‘/test’) {
      transfer(‘123’)
  }
  beforeCallback {
      transfer(‘test transfer’)
  }

  afterCallback {
      println(‘after ’ + transferred()))
  }
```
then `transferred()` will not contain what we passed to `beforeCallback`, but what we put into callback. In this case `"after 123"` is supposed to be printed out

Or am I wrong?

Reply:
I'll definitely check that one out for myself

Reply:
If I don't add `?: nextStepInstance as? T`, it throw a exception when I invoke transferred(). The expcetion is `java.lang.RuntimeException: An instance of class Any is expected, but nothing is stored in chainSource.content`.  It from `BotHanding` 167 line.
I don't know the `transferred()` contain all `transfer` content. Let me check it.

Reply:
I tested using `transfer() with step`, and it overwrites the previous.